readd smsl redirects

taureandyernv opened the pull request:
add back the smsl redirect.  may need additions

ajschmidt8 reviewed (changes requested):
please add redirects to the existing https://github.com/rapidsai/rapids.ai/blob/main/_redirects file.

You can read more about Netlify redirects on their doc page below.

- https://docs.netlify.com/routing/redirects/#syntax-for-the-redirects-file

taureandyernv commented:
thanks @ajschmidt8!  made the changes.  it seems that `/smsl` was the only one without a `/___.html` variant.  Fixed it, deleted the previously created html file, and updated the links

ajschmidt8 commented on _redirects:
```diff
@@ -18,7 +18,8 @@
 /slack-invite          https://join.slack.com/t/rapids-goai/shared_invite/zt-trnsul8g-Sblci8dk6dIoEeGpoFcFOQ
-/smsl                  https://studiolab.sagemaker.aws/import/github/rapidsai-community/rapids-smsl/blob/main/rapids-smsl.ipynb
+/smsl                  https://studiolab.sagemaker.aws/import/github/rapidsai-community/rapids-smsl/blob/main/rapids-smsl.ipynb?ncid=so-othe-849406-vt27#cid=av02_so-othe_en-us
+/smsl.html             https://studiolab.sagemaker.aws/import/github/rapidsai-community/rapids-smsl/blob/main/rapids-smsl.ipynb?ncid=so-othe-849406-vt27#cid=av02_so-othe_en-us
```
```suggestion
```

We don't need a `.html` entry for this redirect.

The other entries only need to include `.html` because of some URL changes that occurred when we switched from Jekyll to Hugo for this site.
